README.md

```diff
@@ -113,6 +177,14 @@ defaults: string|[string]
     - `string`
     - `string: string`
     - `string: [string]`
+```
 
 ## License
 [MIT](LICENSE)
+
+## Author Information
+### Original Author: 
```

Comment:
Rather than specifying original author, how about just listing each of us as an author?